thumbor_shortener/handlers/shortener.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from thumbor_shortener.shortener import Shortener
+from thumbor.handlers.imaging import ImagingHandler
+from thumbor.url import Url
+
+class UrlShortenerHandler(ImagingHandler):
+
+    def get(self, **kwargs):
+
+        # Get the url from the shortener and parse the values.
+        url = Shortener.get(self.request.path)
```

Comment:
For that it could be just a class loaded like the loaders in Thumbor. The context has a "factory" method to create classes using configuration.


For the shortener it might be a good idea to split backends and generators.

ex:
```
# [ generators.Sha256, generators.Md5, generators.ShortMd5, generators.Hashids ]
SHORTENER_GENERATOR=thumbor_shortener.generators.Sha256

# [ backends.Sql, backends.Redis, backends.Memcache ]
SHORTENER_BACKEND=thumbor_shortener.backends.Sql`